GATK uses a Bayesian framework for variant discovery, where the posterior probability P(SNP∣Data) is calculated asP(SNP∣Data)=P(Data∣SNP)⋅P(SNP)/P(Data). We hypothesize that the heterozygosity prior, P(SNP), specified in GATK's HaplotypeCaller may significantly influence the number of SNPs identified, and therefore, the prior probability must be appropriately tuned to the characteristics of the organism under study. Previously, the heterozygosity of *Mimulus laciniatus* was estimated at 0.02, the same value used for *Mimulus guttatus*, an out-crossing species. However, *M. laciniatus* is a self-fertilizing species and is expected to be largely homozygous. Therefore, a heterozygosity estimate of 0.02 may not be appropriate for *M. laciniatus*. Using a more fitting heterozygosity estimate of 0.00555, derived from GenomeScope's k-mer generated spectrum of our assembly, we compare how the heterozygosity prior affects the quality and quantity of SNPs discovered.
