GATK uses a Bayesian framework for variant discovery, where P(SNP∣Data) is computed as P(Data∣SNP)⋅P(SNP)/P(Data). However, outside of this general framework, less is understood about how individual parameters ultimately influence the variants discovered. We hypothesize that the heterozygosity prior, specified in GATK's HaplotypeCaller, may significantly influence SNPs detected, and therefore, the prior probability must be appropriately tuned to the characteristics of the organism under study. Previously, the heterozygosity of *Mimulus laciniatus* was estimated at 0.02, the same value used for out-crossing species *Mimulus guttatus*. However, *M. laciniatus* is a self-fertilizing species and is expected to be largely homozygous. Therefore, a heterozygosity estimate of 0.02 may not be appropriate for *M. laciniatus*. Using a more fitting heterozygosity estimate of 0.00555, derived from GenomeScope's k-mer generated spectrum of our *M. laciniatus* assembly, we compare how the heterozygosity prior affects the quality and quantity of SNPs detected.
